sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SymbolicExecution/TooManyInternalStatesException.cs

```diff
@@ -20,7 +20,7 @@
 
 using System;
 
-namespace SonarAnalyzer.Helpers.FlowAnalysis.Common
+namespace SonarAnalyzer.SymbolicExecution
 {
     public class TooManyInternalStatesException : Exception

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this implementation of 'ISerializable' to conform to the recommended serialization pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3925)